recbole/data/utils.py

```diff
@@ -20,6 +20,7 @@
 from recbole.sampler import KGSampler, Sampler, RepeatableSampler
 from recbole.utils import ModelType
 from recbole.data.dataloader import *
+from recbole.data.dataloader.user_dataloader import *
```

Comment:
```suggestion
```

Please refer to https://github.com/RUCAIBox/RecBole/blob/5b0e4a7c4080ce030f561a864bc5095b2403a7a1/recbole/data/dataloader/__init__.py#L1-L6